src/CheckBoxGroup/CheckBoxGroup.tsx

```diff
@@ -6,7 +6,6 @@ import { CheckBox } from './CheckBox'
 type Item = {
   id: number
   label: string
-  value: string
```

Comment:
Why are we removing the value?

Reply:
it's been there since the beginning but it's never used or passed down the checkbox. If you think make more sense I can pass it down to the input itself?

Reply:
Just wanted to make sure but it makes sense to remove it if it's not used 👍